Demo/Demo/Extensions/UIVIewController+Extensions.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+
+extension UIViewController {
```

Comment:
Also I don't think this should live as an extension to `UIViewController`. It's a bit cleaner to have individual subclasses of UIButton, UITextField, etc such as `CustomPayPalButton`, that customize these certain settings/properties.